src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4156,3 +4156,15 @@ OBA:2060170		UBERON:0002101	limb	PATO:0000140	position
 OBA:2060171		UBERON:0000966	retina	PATO:0000141	structure
 OBA:2060172		UBERON:0005056	external female genitalia	PATO:0000070	amount
 OBA:2060173		UBERON:0000970	eye	PATO:0000070	amount
+OBA:2060179	body coordination	UBERON:0000468	multicellular organism	PATO:0000188	coordination
+OBA:2060180	limb coordination	UBERON:0002101	limb	PATO:0000188	coordination
+OBA:2060181	upper limb coordination	UBERON:0002102	forelimb	PATO:0000188	coordination
+OBA:2060182	lower limb coordination	UBERON:0002103	hindlimb	PATO:0000188	coordination
+OBA:2060183	motor coordination	UBERON:0025525	motor system	PATO:0000188	coordination
```

Comment:
There is a similar [NBO:0000591 motor coordination phenotype](https://www.ebi.ac.uk/ols/ontologies/NBO/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNBO_0000591)

Reply:
We are about to move all of these to OBA, so lets keep a mental note of that!